packages/@webex/plugin-meetings/test/unit/spec/multistream/remoteMedia.ts

```diff
@@ -222,4 +224,24 @@ describe('RemoteMedia', () => {
       });
     });
   });
+
+  describe('setSizeHint()', () => {
+    forEach(
+      [
+        {height: 134, fs: 60},
+        {height: 135, fs: 240},
+        {height: 269, fs: 240},
+        {height: 270, fs: 920},
+        {height: 539, fs: 920},
+        {height: 540, fs: 3600},
+      ],
+      ({height, fs}) => {
+        it(`sets the max fs to ${fs} correctly when height is ${height}`, () => {
+          remoteMedia.setSizeHint(100, height);
+
+          assert.calledOnceWithExactly(fakeReceiveSlot.setMaxFs, fs);
```

Comment:
Do you think there should be a test when receiveSlot is undefined, that is then it is not called

Reply:
I've added a test for this